.github/workflows/spec-digest-gate.yml

```
@@ -0,0 +1,82 @@
+jobs:
+  spec-digest-gate:
+    name: Spec Digest — Proof of Spec Reading
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Detect milestone ID from branch name
+        id: milestone
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)